tests/nvme/047

```diff
@@ -45,6 +45,7 @@ test() {
 		--nr-write-queues 1 \
 		--nr-poll-queues 1 || echo FAIL
 
+	udevadm settle
```

Comment:
`_nvme_connect_subsys` does already do a `udevadm settle` including `_nvme_ctrl_ready`, so `/dev/nvme0` should be available there.

I think the problem is that the namespace device is not yet there. Could you try if 

```bash
_nvmf_wait_for_ns "${def_subsys_uuid}" created
```

would also work here?

Reply:
Though I think we should try to get this into `_nvme_connect_subsys`. It is not a great that we have to start sprinkling these calls around in the tests.